api/user/login.py

```
@@ -3,38 +3,38 @@
-    key = "Uemail" if post.exists("Uemail") else "Uname"
     Uid, Uhashpass = database.request(
-        "SELECT Uid, Uhashpass FROM USERS WHERE ! = ?",
+        "SELECT Uid, Uhashpass FROM USERS WHERE Uname = ?",
         [
-            key,
-            SUG.REQ.PST[key]
+            SUG.REQ.PST["Uname"]
         ]
-    )[0]
+    )[0].values()
+    debug(Uhashpass, SUG.REQ.PST["Uhashpass"], Uid, database.request(
```

Ruff (0.11.9)

`debug` may be undefined, or defined from star imports

(F405)